scripts/build_awslc.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+# Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Released under Apache 2.0 license as described in the file LICENSE.
+# Author(s): Yan Peng
+
+set -e
+set -o xtrace
+
+# Remember where LNSym is
+LNSym_DIR=${PWD}
+echo $CC
+echo $CXX
+clang --version
+gcc --version
+
+echo ""
+echo "Environment variables:"
+env
+
+# Install dependencies
+# brew install ninja
+
+# Fetching AWS-LC
+git clone https://github.com/aws/aws-lc.git $HOME/aws-lc --depth 1
+cd $HOME/aws-lc
+# Build AWS-LC
+mkdir aws-lc-build; cd aws-lc-build
+# cmake -GNinja -DKEEP_ASM_LOCAL_SYMBOLS=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo ../
+cmake -DKEEP_ASM_LOCAL_SYMBOLS=1 -DCMAKE_BUILD_TYPE=RelWithDebInfo ../
```

Comment:
```suggestion
cmake  -DCMAKE_BUILD_TYPE=RelWithDebInfo ../
```

It seems to build locally (and I can run the file `crypto/crypto_test`) for me without this flag.